llvm/bindings/ocaml/debuginfo/debuginfo_ocaml.c

```diff
@@ -969,45 +969,6 @@ value llvm_dibuild_create_parameter_variable_bytecode(value *argv, int arg) {
   );
 }
 
-value llvm_dibuild_insert_declare_before_native(value Builder, value Storage,
-                                                value VarInfo, value Expr,
-                                                value DebugLoc, value Instr) {
-  LLVMDbgRecordRef Value = LLVMDIBuilderInsertDeclareBefore(
```

Comment:
I think we want to keep these functions and change `LLVMDIBuilderInsertDeclareBefore` for the record versions (same below).

Reply:
Restoring the deleted functions and changed them to use the record versions.